tests/integration/common/s2n_test_reporting.py

```diff
@@ -0,0 +1,58 @@
+##
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+"""
+Common functions used to report the results of tests.
+"""
+
+import sys
+from enum import Enum
+
+
+class Status(Enum):
+    PASSED = 32
+    FAILED = 31
```

Comment:
It's not obvious to me why these values are 32 & 31

Reply:
It's the value that represents the color they print as :) I could store that information separately, but since PASSED and FAILED just need unique values, using the colors seemed convenient. Should I add a comment?